examples/mlserver/client.py

```diff
@@ -0,0 +1,27 @@
+import requests, threading
+
+NUM_THREADS = 2
+URL = "http://localhost:8080/v2/models/text-classification-model/infer"
+sentences = ["I hate using GPUs for inference", "I love using DeepSparse on CPUs"] * 100
+
+def tfunc(text):
+    inference_request = {
+        "inputs": [
+            {
+                "name": "sequences",
+                "shape": [1],
+                "datatype": "BYTES",
+                "data": [text],
+            },
+        ]
+    }   
+    resp = requests.post(URL, json=inference_request).json()
+    for output in resp["outputs"]:
+        print(output["data"])
+
+
+threads = [threading.Thread(target=tfunc, args=(sentence,)) for sentence in sentences[:NUM_THREADS]]
+for thread in threads:
+    thread.start()
+for thread in threads:
+    thread.join()
```

Comment:
it looks like this creates `NUM_THREADS` threads to make the request, is that intended?
Might make more sense to create `len(sentences)` threads and execute `NUM_THREADS` at a time.

You can do this out of the box with `ThreadPoolExecutor` with something like:

```suggestion
from concurrent.futures.thread import ThreadPoolExecutor

threadpool = ThreadPoolExecutor(max_workers=NUM_THREADS)

results = threadpool.map(tfunc, sentences)
```